src/invidious/helpers/youtube_api.cr

```diff
@@ -89,6 +89,33 @@ def request_youtube_api_search(search_query : String, params : String, region =
   return _youtube_api_post_json("/youtubei/v1/search", data)
 end
 
+####################################################################
+# request_youtube_api_next(continuation)
+#
+# Requests the youtubei/v1/next endpoint with the required headers
+# and POST data in order to get a JSON reply in english that can
+# be easily parsed.
+#
+#
+# The requested data can either be:
+#
+#  - A continuation token (ctoken). Depending on this token's
+#    contents, the returned data can be comments, playlist videos,
+#    search results, channel community tab, ...
+#
+#  - A playlist ID (parameters MUST be an empty string)
+#
+def request_youtube_api_next(continuation : String, playlistId : String)
+  # JSON Request data, required by the API
+  data = {
+    "context"    => make_youtube_api_context("US"),
+    "videoId"    => continuation,
+    "playlistId" => playlistId,
+  }
```

Comment:
The `continuation` parameter needs to be renamed to something along the lines of `video_id`. Please also update the documentation for it!

Reply:
This part will be overriden by #2217 and #2220 anyway (as those will be merged first).

Reply:
done!

Reply:
> This part will be overriden by #2217 and #2220 anyway (as those will be merged first).

Should they be merged first? I personally think important bug fixes like this should get into master as soon as possible.

Reply:
The problem is that we currently have 3 PRs that implements `request_youtube_api_next`, and I'd like to do something in order to not have to deal with that later (i.e: I'm thinking about implementing `next` and `player` endpoints now and some code cleaning)

Reply:
Good idea to just add the endpoints in the code in a separate PR.

Reply:
done